tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -155,11 +158,15 @@ public function testDropPrimaryKeyWithAutoincrementColumn()
      */
     public function testDoesNotPropagateDefaultValuesForUnsupportedColumnTypes()
     {
+        if ($this->_sm->getDatabasePlatform() instanceof MariaDb102Platform) {
```

Comment:
This should be based on `$type->supportsDefaultValue($platform)`, not based on the class name.

Reply:
> Let's first decide if it's required as part of initial MariaDB support (see #2825 (comment)). MySQL doesn't support them (not sure about other platforms). Why introduce them for MariaDB?

Let's decide later